paddlenlp/data/tokenizer.py

```diff
@@ -37,6 +37,15 @@ def encode(self, sentence):
 
 
 class JiebaTokenizer(BaseTokenizer):
+    """
+    Constructs an tokenizer based on `jieba <https://github.com/fxsjy/jieba>`__. 
+    It supports :meth:`cut` method to cut the text to tokens, and :meth:`encode` 
+    method to covert text to ids.
```

Comment:
cut the text -> split the text

Reply:
covert text to ids. -> covert text to token ids.

Reply:
Done. Thanks.